### ISSUE TITLE
commentitem doesn't render object

### ISSUE THREAD

**Issue by dormied** [GetStream/react-native-activity-feed#41]
**Setup (always fill this in):**

- iOS or Android? ios
- Expo or regular React Native? Expo
- React Native Version:
expo-activity-feed@0.8.8
- `getstream` version: `npm list --pattern getstream`/`yarn list --pattern getstream`
getstream@4.2.1
**Describe the bug**
When using the SinglePost component, comment objects do not appear (but evrything else does). I went into react-native-activity-feed-core module and changed the comment.data.text line to comment.data.data.text and the comment shows up.

            _react2.default.createElement(
              _reactNative.Text,
              { style: styles.commentContent, __source: {
                  fileName: _jsxFileName,
                  lineNumber: 47
                }
              },
              **comment.data.data.text**,
              ' '
            ),

I printed out the activity object to the console for the activity props that gets passed into the SinglePost component and you can see that the comment object is nested in two data objects.

Object {
[20:00:39]   "actor": Object {
[20:00:39]     "created_at": "2019-03-10T17:39:17.184729Z",
[20:00:39]     "data": Object {
[20:00:39]       "email": "mail@robertly.com",
[20:00:39]       "name": "Robert Ly",
[20:00:39]       "profile": Object {
[20:00:39]         "city": "",
[20:00:39]         "dtlVideoUrl": "",
[20:00:39]         "foVideoUrl": "",
[20:00:39]         "image": "",
[20:00:39]       },
[20:00:39]     },
[20:00:39]     "id": "tcORzOvprJTE8831hqESd1QP3tW2",
[20:00:39]     "updated_at": "2019-03-10T17:39:17.184729Z",
[20:00:39]   },
[20:00:39]   "foreign_id": "",
[20:00:39]   "id": "7f4136be-446d-11e9-a74b-127939012af0",
[20:00:39]   "object": "Hey check out my swing",
[20:00:39]   "origin": null,
[20:00:39]   "own_reactions": Object {
[20:00:39]     "comment": Array [
[20:00:39]       Object {
[20:00:39]         "activity_id": "7f4136be-446d-11e9-a74b-127939012af0",
[20:00:39]         "children_counts": Object {},
[20:00:39]         "created_at": "2019-03-12T02:37:15.823271Z",
**[20:00:39]         "data": Object {
[20:00:39]           "data": Object {
[20:00:39]             "text": "My comment!",
[20:00:39]           },
[20:00:39]         },**
[20:00:39]         "id": "0d92cd51-a455-4a64-9623-13ee56ef476b",
[20:00:39]         "kind": "comment",
[20:00:39]         "latest_children": Object {},
[20:00:39]         "parent": "",
[20:00:39]         "updated_at": "2019-03-12T02:37:15.823271Z",
[20:00:39]         "user": Object {
[20:00:39]           "created_at": "2019-03-12T01:03:31.590490Z",
[20:00:39]           "data": Object {
[20:00:39]             "email": "anotheruser@gmail.com",
[20:00:39]             "name": "another user",
[20:00:39]             "profile": Object {
[20:00:39]               "city": "",
[20:00:39]               "dtlVideoUrl": "",
[20:00:39]               "foVideoUrl": "",
[20:00:39]               "image": "",
[20:00:39]             },
[20:00:39]           },
[20:00:39]           "id": "prbd9f3xRhh5a98gAq46gm9IEhz2",
[20:00:39]           "updated_at": "2019-03-12T01:03:31.590490Z",
[20:00:39]         },
[20:00:39]         "user_id": "prbd9f3xRhh5a98gAq46gm9IEhz2",
[20:00:39]       },
[20:00:39]     ],
[20:00:39]   },
[20:00:39]   "reaction_counts": Object {
[20:00:39]     "comment": 1,
[20:00:39]   },
[20:00:39]   "target": "",
[20:00:39]   "time": "2019-03-12T02:21:08.861511",
[20:00:39]   "verb": "post",
[20:00:39] }

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by JelteF** [comment]
What code did you use to create this reaction? Looking at the source code, the built in CommentBox should not do this double data behaviour.

**Comment by dormied** [comment]
Yeah, I'm using CommentBox. I'm not using anything custom at the moment other than the provided RN components.

**Comment by jaapbakker88** [comment]
Is this still an issue. If yes, could you share the code from the CommentBox. The issue is probably there.

**Comment by familywiser1** [comment]
I was also having this problem then I noticed the problem is with example code and not with CommentItem. 

I think you should fix the example code at - https://github.com/GetStream/react-native-example/blob/master/screens/SinglePostScreen.js#L85

```
<CommentBox
                onSubmit={(text) =>
                  props.onAddReaction('comment', activity, {
                    //this is wrong - it creates a nested data.data.text object
                   data: { text: text },
                  })
                }
                avatarProps={{
                  source: (userData: UserResponse) =>
                    userData.data.profileImage,
                }}
                styles={{ container: { height: 78 } }}
              />
```
Here is correct example:

```
<CommentBox
                                                onSubmit={(text) =>
                                                    props.onAddReaction('comment', activity, {text: text})
                                                }
                                                avatarProps={{
                                                    source: (userData) =>
                                                        userData.data.profileImage,
                                                }}
                                                styles={{container: {height: 78}}}
                                            />
```
